src/codeu/chat/common/Bot.java

```diff
@@ -0,0 +1,13 @@
+package codeu.chat.common;
+
+public abstract class Bot {
```

Comment:
JavaDoc comments would help people creating Bots understand what these functions are supposed to do